docs/installing-solidity.rst

```diff
@@ -228,6 +228,27 @@ in Visual Studio 2017 Build Tools or Visual Studio 2017:
 .. _Visual Studio 2017: https://www.visualstudio.com/vs/
 .. _Visual Studio 2017 Build Tools: https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017
 
+Prerequisites - OpenBSD
+-----------------------
+
+You need to install the following dependencies for OpenBSD builds of Solidity:
+
++-----------------------------------+-------------------------------------------------------+
+| Software                          | Notes                                                 |
++===================================+=======================================================+
+| `CMake`_                          | Cross-platform build file generator.                  |
++-----------------------------------+-------------------------------------------------------+
+| `Boost`_                          | C++ libraries.                                        |
++-----------------------------------+-------------------------------------------------------+
+| `z3`_                             | Theorem prover.                                       |
```

Comment:
I wouldn't say this is a prereq.

Reply:
@axic BSD doesn't come with any of these packages, and you need them to build Solidity, so that's a requirement isn't it?

Reply:
It's an optional dependency.

Reply:
Sorry @axic and @chriseth from the line comment I can't tell which dependency you're referring to. Does this mean that it's also optional for Linux, macOS etc too?

Reply:
z3 is optional on all platforms. It is not listed on the others, shouldn't list it for BSD. Unless the `install_deps` script is broken and requires it, of course.

Reply:
@axic Ah ha, OK, that was just taken from the original poster. And what do you think about adding the "untested" disclaimer?